i18n.ts

```diff
@@ -43,3 +48,55 @@ function getLanguageCode(code: string) {
   const [language] = code.split('-');
   return language;
 }
+
+export function getVCDetailsForCurrentLanguage(locales) {
+  const supportedLanguages = Object.keys(SUPPORTED_LANGUAGES);
+  const currentLanguage = i18next.language;
+  for (const index in supportedLanguages) {
+    const supportedLanguage = supportedLanguages[index];
+    if (supportedLanguage == currentLanguage) {
+      const vcDetailsForCurrentLanguage = locales.filter(
+        (obj) => obj.language === languageCodeMap[currentLanguage]
+      );
+      return vcDetailsForCurrentLanguage[0]?.value
+        ? vcDetailsForCurrentLanguage[0].value
+        : locales[0]?.value;
+    }
+  }
+}
+
+// This method gets the value from iso-639-3 package, which contains key value pairs of three letter language codes[key] and two letter langugae code[value]. These values are according to iso standards.
+// The response recieved from the server is three letter language code and the value in the inji code base is two letter language code. Hence the conversion is done.
+function getThreeLetterLanguageCode(twoLetterLanguageCode) {
+  let threeLetterLanguageCode = Object.keys(iso6393To1).find(
+    (key) => iso6393To1[key] === twoLetterLanguageCode
+  );
+  if (!threeLetterLanguageCode) {
+    threeLetterLanguageCode = Object.keys(iso6393To2T).find(
```

Comment:
why are we using two different standards?

Reply:
In the supported languages list we have three letter code and two letter language codes. In this method getThreeLetterLanguageCode we pass all the supported languages and generate the three letter code. iso6393To1 list has key as 3 letter language code and value as 2 letter language code. If we pass 3 letter language code to iso6393To1 it return undefined. iso6393To2B has key and value as 3 letter language code. Hence we are using two different standards.